packages/rum/src/boot/lazyLoadRecorder.ts

```diff
@@ -0,0 +1,3 @@
+export function lazyLoadRecorder() {
+  return import(/* webpackChunkName: "recorder" */ './startRecording').then((module) => module.startRecording)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ I have a doubt, does this means that customer need to use webpack as well? or is this a stupid question?

Reply:
Not exactly, I used the `webpackChunkName` special comment to have a fixed chunk name to deploy our CDN files. For customers using NPM:
- if they use webpack: the chunk will have a fix name
- If they use another bundler this comment is ignored

Reply:
I don't think we should use a fixed name for chunks though. It should include a hash or something to avoid cache issues, because I don't trust cloudfront/the browser to invalidate both files exactly at the same time. 

We can talk about it.

Reply:
Good point! I changed the chunks format to be 
https://github.com/DataDog/browser-sdk/blob/1acd4ac7309093080d6fb6a0d0efaccb1ae443cf/webpack.base.js#L14